test/JDBC/input/BABEL-INSERT-EXECUTE.sql

```diff
@@ -242,6 +242,33 @@ insert into @a execute('select * from t1; select 3');
 select * from @a;
 go
 
+create schema user_defined_sch;
```

Comment:
I think we should check cross db scenarios as well.

Reply:
I see, make sense